src/jit/codegenxarch.cpp

```diff
@@ -5172,16 +5172,17 @@ void CodeGen::genTransferRegGCState(regNumber dst, regNumber src)
 //     pass in 'addr' for a relative call or 'base' for a indirect register call
 //     methHnd - optional, only used for pretty printing 
 //     retSize - emitter type of return for GC purposes, should be EA_BYREF, EA_GCREF, or EA_PTRSIZE(not GC)
```

Comment:
If we consider this PR as addressing gc-ness of structs passed/returned in two registers, then the fix to above issue can be included with this PR.  On the other hand I am equally fine if @LLITCHEV decides to file an issue for it and address it with a separate check-in.